fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/observer/FedoraEvent.java

```diff
@@ -64,7 +61,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
-        eventID = pidMinter.get();
+        eventID = randomUUID().toString();
```

Comment:
Instead of hard-coding `randomUUID()` here, is there a reason why an `UniqueValueSupplier<String>`, as suggested in the code review comments, was not injected?

As a note, `randomUUID().toString()` is not equivalent to the previous `UUIDPathMinter.get()`. `UUIDPathMinter` provides a pairtree path with a UUID leaf element, whereas `randomUUID` simply provides a UUID.
The reason a full pairtree path is important is because the `InternalAuditor` uses the `FedoraEvent#eventID` as the identifier name at which an audit record resource is created:
https://github.com/fcrepo4-labs/fcrepo-audit/blob/master/src/main/java/org/fcrepo/audit/InternalAuditor.java#L211

The pairtree path provided by the `UUIDPathMinter` gets around the performance limitation of too many children of a given resource.